css/app.css

```diff
@@ -35,3 +35,28 @@ gaia-list .destructive {
 a.navigation *, a.action *, a.edit * {
   pointer-events: none;
 }
+
+gaia-list span.group {
+  text-transform: capitalize;
+  font-size: 0.8em;
```

Comment:
It's not.
Those are more akin to the letter "A", "B", "C"... in your contact list. Which are smaller than the contact items themselves.